alacritty/src/event.rs

```diff
@@ -1078,6 +1079,17 @@ impl<N: Notify + OnResize> Processor<N> {
     }
 }
 
+fn start_program<I, S>(program: &str, args: I)
+where
+    I: IntoIterator<Item = S> + Debug + Copy,
+    S: AsRef<OsStr>,
+{
+    match start_daemon(program, args) {
```

Comment:
Do we not want to provide this functionality globally in `start_daemon`? Are there instances where we do not want to print a warning?

Reply:
I'm not sure that we should do that in `start_daemon`. It does the one thing, and have a special impl on each platform, so I'm not sure if there's any value in moving it there. Hiding errors from library users isn't something I really like tbqh.

Reply:
Could you move it out of `alacritty_terminal` then maybe? There's no reason for it to be there.